charts/monitoring/grafana/dashboards/kubernetes/kubelets.json

```diff
@@ -2295,5 +966,5 @@
   "timezone": "",
   "title": "Kubelets",
   "uid": "p4ynSSeik",
-  "version": 6
+  "version": 1
```

Comment:
I think this should be `7` as it's updating an existing dashboard. Grafana expects the `version` field to be incremented only, I can see this causing problems when loading the dashboard.